tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -22,13 +24,22 @@ def get_arguments():
     2) a "class file" (newline delimited list of strings that correspond to RDF classes)
     """
     parser = argparse.ArgumentParser(description='Process data model export')
-    parser.add_argument('-f', '--file_path', help="a path to the data model", required=True)
+    parser.add_argument('-f', '--file_path', help="a path to the data model")
     # TODO: two methods to input classes, CLI list vs CLI file_path argument - keep both?
     parser.add_argument('-l', '--class_list', nargs='+', help="a class listing string")
     parser.add_argument('-c', '--class_path', help="a class listing file")
```

Comment:
nit: use `kebab-case` rather than `snake_case` when specifying command-line argument names for consistency with other command line programs. the variables will still be accessible in your python code in their snake case format.